yaksh/static/yaksh/js/requesthandler.js

```diff
@@ -126,19 +142,26 @@ $(document).ready(function(){
           render();
       }
   };
-
+  if (question_type == 'code'){
+  
   // Initialize the codemirror editor
   global_editor.editor = CodeMirror.fromTextArea(textarea_node, options);
-
   // Setting code editors initial content
   global_editor.editor.setValue(init_val);
-
+}
+if (question_type == 'upload' || question_type == 'code') {
   $('#code').submit(function(e) {
     lock_screen();
+    if (question_type == "code"){
     var data = $(this).serializeArray();
+  }
+  else if (question_type == "upload"){
+    var data = new FormData($("#code").get(0));
```

Comment:
Just a suggestion.
Instead of get(),  we can pass form using document.getElementById, just for clarity.